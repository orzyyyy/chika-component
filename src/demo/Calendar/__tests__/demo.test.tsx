import React from 'react';
import { shallow } from 'enzyme';
import 'nino-cli/scripts/setup';
import CalendarDemo from '..';

describe('CalendarDemo', () => {
  it('render correctly', () => {
    const wrapper = shallow(<CalendarDemo />);
    expect(wrapper).toMatchSnapshot();
  });
});
