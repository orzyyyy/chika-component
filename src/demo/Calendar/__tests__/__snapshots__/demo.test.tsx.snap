// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CalendarDemo render correctly 1`] = `
<div
  className="Calendar_demo"
>
  <div
    onClick={[Function]}
    style={
      Object {
        "float": "left",
      }
    }
  >
    上个月
  </div>
  <div
    onClick={[Function]}
    style={
      Object {
        "float": "right",
      }
    }
  >
    下个月
  </div>
  <div
    style={
      Object {
        "clear": "both",
      }
    }
  />
  <div>
    2019/4/1 2019/4/30
  </div>
  <Calendar
    end="2019/4/30"
    onChange={[Function]}
    position=""
    select={Array []}
    start="2019/4/1"
    touch={[Function]}
  />
</div>
`;
